src-browser/process.js

```diff
@@ -1,25 +1,210 @@
-import process from '../node_modules/process/browser.js';
-process.platform = 'browser';
-export default process;
-export var addListener = process.addListener;
-export var argv = process.argv;
-export var binding = process.binding;
-export var browser = process.browser;
-export var chdir = process.chdir;
-export var cwd = process.cwd;
-export var emit = process.emit;
-export var env = process.env;
-export var listeners = process.listeners;
-export var nextTick = process.nextTick;
-export var off = process.off;
-export var on = process.on;
-export var once = process.once;
-export var prependListener = process.prependListener;
-export var prependOnceListener = process.prependOnceListener;
-export var removeAllListeners = process.removeAllListeners;
-export var removeListener = process.removeListener;
-export var title = process.title;
-export var umask = process.umask;
-export var version = process.version;
-export var versions = process.versions;
+function unimplemented(name) {
+  throw new Error('Node.js process ' + name + ' is not supported by JSPM core outside of Node.js');
+}
+
+export var nextTick = queueMicrotask.bind(self);
```

Comment:
Per https://nodejs.org/dist/latest-v16.x/docs/api/process.html#process_when_to_use_queuemicrotask_vs_process_nexttick it seems like a promise-based approach may be closer to `nextTick` here:

```suggestion
export var nextTick = cb => Promise.resolve().then(cb);
```